install/installer/aws.py

```diff
@@ -0,0 +1,272 @@
+        keys.append(key['AccessKeyId'])
+
+    logger.debug(f'User access keys: {keys}')
+    return keys
+
+
+@typechecked
+def check_affiliation_user_and_creds(
+    user: str,
+    login: str,
+    password: str,
+    keys: list,
+) -> Dict[str, Union[str, bool]]:
+    """Check that provided keys affiliate to provided username of exist user.
+
+    If specified user exist, --cloud-programatic-login and
+    --cloud-programatic-password will be try use as it.
+
+    Args:
+        user: (str) Cluster.dev user name.
+        login: (str) Cloud programatic login.
+        password: (str) Cloud programatic password.
+        keys: (list) AWS_ACCESS_KEY_ID's
+    Returns:
+        Dict[str, Union[str, bool]]:
+        - `{'key': 'AWS_ACCESS_KEY_ID', 'secret': 'AWS_SECRET_ACCESS_KEY', 'created': bool}`
+        - `{}` - if keys not affiliate to user.
+    """
+    for key in keys:
+        if key == login:
+            logger.info(f'Specified creds belong to user "{user}", use it')
+
+            return {
+                'key': login,
+                'secret': password,
+                'created': False,
+            }
+
+    logger.debug(f'Specified keys not belong to user "{user}"')
+    return {}
+
+
+@typechecked
+def create_access_key(
+    iam: boto3.client,
+    user: str,
+) -> Dict[str, Union[str, bool]]:
+    """Create Access Key for exist user.
+
+    Required "iam:CreateAccessKey".
+
+    Args:
+        iam: (boto3.client.IAM) A low-level client representing AWS IAM.
+        user: (str) Cluster.dev user name.
+
+    Returns:
+        Dict[str, Union[str, bool]]:
+        `{'key': 'AWS_ACCESS_KEY_ID', 'secret': 'AWS_SECRET_ACCESS_KEY', 'created': bool}`
+    """
+    response = iam.create_access_key(UserName=user)
+
+    logger.debug('Access keys created')
+    return {
+        'key': response['AccessKey']['AccessKeyId'],
+        'secret': response['AccessKey']['SecretAccessKey'],
+        'created': True,
+    }
+
+
+@typechecked
+def create_user_and_permissions(
+    user: str,
+    login: str,
+    password: str,
+    session: str,
+    release: str,
+) -> Dict[str, Union[str, bool]]:
+    """Create cloud user and attach needed permissions.
+
+    Args:
+        user: (str) Cluster.dev user name.
+        login: (str) Cloud programatic login.
+        password: (str) Cloud programatic password.
+        session: (str) Cloud session token.
+        release: (str) cluster.dev release tag.
+
+    Returns:
+        Dict[str, Union[str, bool]]:
+        `{'key': 'AWS_ACCESS_KEY_ID', 'secret': 'AWS_SECRET_ACCESS_KEY', 'created': bool}`.
+    """
+    iam = boto3.client(
+        'iam',
+        aws_access_key_id=login,
+        aws_secret_access_key=password,
+        aws_session_token=session,
+    )
+
+    # cli_rights_needed_msg
+    msg = (
+        '\n\nRights what you need described here: \n'
+        + f'https://github.com/shalb/cluster.dev/blob/{release}'
+        + '/install/installer_aws_install_req_permissions.json'
```

Comment:
Concat is bad
```suggestion
        '\n\nRights what you need described here: \n'
        f'https://github.com/shalb/cluster.dev/blob/{release}'
        '/install/installer_aws_install_req_permissions.json'
```

Reply:
Concat is good

WPS326 Found implicit string concatenation
https://wemake-python-stylegui.de/en/latest/pages/usage/violations/consistency.html#wemake_python_styleguide.violations.consistency.ImplicitStringConcatenationViolation